src/styles/mo-components/_buttons.scss

```diff
@@ -1,7 +1,17 @@
 button {
   &.btn {
+    font-size: 1em;
+    padding: .375rem .75rem;
```

Comment:
I'm not sure if this will mess up anyone's buttons. I'm hoping their styles would be more specific. Like https://github.com/MoveOnOrg/giraffe/blob/main/src/styles/components/_form.scss#L367

Reply:
I pulled and built this branch and don't see any buttons in the comps regressing, but it would be good to give everyone a heads up in the channel. These generic buttons ought to match or replace what's in the styleguide at static.moveon.org/giraffe/styleguide.html.